bild.py

```diff
@@ -42,6 +43,8 @@
 PYTHON3_TARGET	= "../antlr4-python3"
 CSHARP_TARGET	= "../antlr4-csharp"
 
+JAVA_VERSION = os.environ.get('JAVA_VERSION', None)
```

Comment:
We have no need for java 1.7 right now,
although we should start advertising that next major release will move to 1.7

> Le 11 janv. 2015 à 10:54, Terence Parr <notifications@github.com> a écrit :
> 
> In bild.py <https://github.com/antlr/antlr4/pull/748#discussion-diff-22763847>:
> 
> > @@ -42,6 +43,8 @@
> >  PYTHON3_TARGET	= "../antlr4-python3"
> >  CSHARP_TARGET	= "../antlr4-csharp"
> >  
> > +JAVA_VERSION = os.environ.get('JAVA_VERSION', None)
> ok, javac is piece of shit:
> 
> terence:master:~/github/bild $ javac -source 1.6 -target 1.6
> javac: no source files
> Usage: javac <options> <source files>
> use -help for a list of possible options
> terence:master:~/github/bild $ javac -source 1.7 -target 1.6
> javac: source release 1.7 requires target release 1.7
> why have -target if the damn thing won't gen earlier versions. We have 1.7 code in some places, right?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/antlr/antlr4/pull/748/files#r22763847>.
>